server/src/main/java/org/opensearch/wlm/WorkloadManagementSettings.java

```diff
@@ -113,6 +167,57 @@ public WorkloadManagementSettings(Settings settings, ClusterSettings clusterSett
         clusterSettings.addSettingsUpdateConsumer(NODE_LEVEL_MEMORY_REJECTION_THRESHOLD, this::setNodeLevelMemoryRejectionThreshold);
         clusterSettings.addSettingsUpdateConsumer(NODE_LEVEL_CPU_CANCELLATION_THRESHOLD, this::setNodeLevelCpuCancellationThreshold);
         clusterSettings.addSettingsUpdateConsumer(NODE_LEVEL_CPU_REJECTION_THRESHOLD, this::setNodeLevelCpuRejectionThreshold);
+        clusterSettings.addSettingsUpdateConsumer(WLM_MODE_SETTING, this::setWlmMode);
+        clusterSettings.addSettingsUpdateConsumer(QUERYGROUP_SERVICE_RUN_INTERVAL_SETTING, this::setQueryGroupServiceRunInterval);
+        clusterSettings.addSettingsUpdateConsumer(QUERYGROUP_SERVICE_DURESS_STREAK_SETTING, this::setDuressStreak);
+    }
+
+    /**
+     * node duress streak getter
+     * @return current duressStreak value
+     */
+    public int getDuressStreak() {
+        return duressStreak;
+    }
+
+    /**
+     * node duress streak setter
+     * @param duressStreak new value
+     */
+    public void setDuressStreak(int duressStreak) {
+        this.duressStreak = duressStreak;
+    }
```

Comment:
nit: Do these setters need to be public?

Reply:
Agree! this doesn't need to be public but had seen the setting consumers to be public